.github/workflows/scanner-dev-vuln-update.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
       (github.event_name == 'pull_request' &&
        contains(github.event.pull_request.labels.*.name, 'pr-update-scanner-vulns'))
     container:
-      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.4.2
+      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.4.3
```

Comment:
just realized this is `stackrox-test` instead of `scanner-test`. Can you switch it?